src/extractMeta.ts

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable security/detect-object-injection */
 import { fromCamelToSentence } from './utils';
 
-const extractMetadata = (props = {}) => {
-    const extraction = {};
-    Object.keys(props).forEach(key => {
+const extractMetadata = (props: any = {}) => {
+    const extraction: Record<string, any> = {};
```

Comment:
@divinentd would this not be preferable?

```suggestion
const extractMetadata = (props: Record<string, unknown> = {}) => {
    const extraction: Record<string, unknown> = {};
```

Reply:
Yes, good point.